ext/pdo_sqlite/sqlite_statement.c

```diff
@@ -246,10 +247,19 @@ static int pdo_sqlite_stmt_describe(pdo_stmt_t *stmt, int colno)
 	stmt->columns[colno].precision = 0;
 
 	switch (sqlite3_column_type(S->stmt, colno)) {
+		case SQLITE_BLOB:
+			if (stmt->bound_columns && (
+					(param = zend_hash_index_find_ptr(stmt->bound_columns, colno)) != NULL ||
+					(param = zend_hash_find_ptr(stmt->bound_columns, stmt->columns[colno].name)) != NULL)) {
+
+				if (PDO_PARAM_TYPE(param->param_type) == PDO_PARAM_LOB) {
+					stmt->columns[colno].param_type = PDO_PARAM_LOB;
+					break;
+				}
+			}
```

Comment:
I think a comment like `/* fall through */` is appropriate here to clarify that the fall through is intended.